tests/test_0884-index-and-identifier-refactoring.py

```diff
@@ -0,0 +1,12 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+from __future__ import absolute_import
+
+import pytest  # noqa: F401
+import numpy as np  # noqa: F401
+import awkward as ak  # noqa: F401
+
+
+def test():
+    # FIXME
+    pass
```

Comment:
This is where you would put tests that compare the Python Index against the C++ Index8, IndexU8, Index32, IndexU32, Index64. If you have more than one test (very likely), the function names must be `test_XYZ` for some `XYZ`. (That's how pytest finds them.)

With this file in place,

```bash
python localbuild.py --pytest tests
```

would include this test and

```bash
python localbuild.py --pytest tests/test_0884-index-and-identifier-refactoring.py
```

would run only this test (faster).